MIGRATING.md

```diff
@@ -299,6 +299,30 @@ major releases of `cosmwasm`. Note that you can also view the
   and 28 bytes when its canonical representation is base64 encoded). For fixed
   length database keys `CanonicalAddr` remains handy though.
 
+- Replace `StakingMsg::Withdraw` with `DistributionMsg::SetWithdrawAddress` and
+  `DistributionMsg::WithdrawDelegatorReward`. `StakingMsg::Withdraw` was a
+  shorthand for the two distribution messages. However, it was unintuitive
+  because it did not set the address for one withdraw only but for all following
+  withdrawls. Since withdrawls are [triggered by different
+  events][distribution docs] such as validators changing their commission rate,
```

Comment:
Missing link to the docs it seems

Reply:
It should be here: https://github.com/cosmos/cosmos-sdk/tree/master/x/distribution/spec (or some sub-page there)

Reply:
I used the `[link text][link id]` syntax with `[link id]: the_url` in the bottom of the section

Reply:
Ah... a true markdown wizard :mage: 

I thought it was a typo. Looks good.